### ISSUE TITLE
Talkback accessibility

### ISSUE THREAD

**Issue by davidbtc2009** [session-foundation/session-android#393]
**Code of conduct**

- [x] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**

Talkback screen reader reads the profile button and search button on the main screen as "unlabeled" and the "+" icon cannot even be focused. It goes from the last chat in the list back to the unlabeled profile button. Once the "+" icon is clicked visually the globe icon, message bubble icon, snd group icons are also not able to be focused in order to be used by talkback. When inside a chat the "+" button and microphone are not usable by talkback as they cannot be focused.

**To reproduce**

1. Open Settings > Accessibility > Talkback and turn it on
2. Open session
3. Swipe through the icons on the main screen
4. Then open a chat and swipe through the chat screen

**Smartphone (please complete the following information):**

 - Device: Motorola One 5G Ace
 - OS: Android 11
 - Version of Session or latest commit hash: 1.11.17 from F-Droid Session repo
